sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -1311,19 +1327,12 @@ Command *Scheduler::GraphBuilder::connectDepEvent(
            Dep.MDepCommand);
     // add user to Dep.MDepCommand is already performed beyond this if branch
 
-    MemObjRecord *Record = getMemObjRecord(Dep.MDepRequirement->MSYCLMemObj);
-    updateLeaves({Dep.MDepCommand}, Record, Dep.MDepRequirement->MAccessMode,
-                 ToCleanUp);
-
     std::vector<Command *> ToEnqueue;
-    addNodeToLeaves(Record, ConnectCmd, Dep.MDepRequirement->MAccessMode,
```

Comment:
Could you, please, elaborate why a command doesn't get added to leaves and the leaves aren't updated?

Reply:
do you mean to leave a comment in code? this code it removed to stop adding connectCmd to the leaves buffer. ConnectCmd is always connected with the new leaf and is always added as dependency to it.

Reply:
> do you mean to leave a comment in code?

Describing it here will do. Thanks.